source/reference/kubectl-minio-plugin/kubectl-minio-tenant-expand.rst

```diff
@@ -92,18 +96,33 @@ The command supports the following flags:
 .. mc-cmd:: --volumes
    :required:
 
-   The number of volumes in the new MinIO Tenant Pool. 
+   Mutually exclusive with :mc-cmd:`~kubectl minio tenant expand --volumes-per-server`.
+   Use either :mc-cmd:`~kubectl minio tenant expand --volumes` or :mc-cmd:`~kubectl minio tenant expand --volumes-per-server`.
+
+The number of volumes in the new MinIO Tenant Pool. 
```

Comment:
Check the indentation at the beginning of the line. Looks like it's missing here.